Common/Orders/MarketOrder.cs

```diff
@@ -42,6 +42,50 @@ public override decimal Value
         }
 
         /// <summary>
+        /// Create update request for pending orders. Null values will be ignored.
+        /// </summary>
+        public UpdateOrderRequest UpdateRequest(int? quantity = null, string tag = null)
```

Comment:
Please update all the `UpdateRequest` functions to require a the time value as a parameter. This needs to be set to the algorithm time.